### ISSUE TITLE
kernel.randomize_va_space is not equal to 0 in 4.1.17-boot2docker

### ISSUE THREAD

**Issue by zavalit** [boot2docker/boot2docker#1136]
Hi guys, after 3 days debuging, i finally  found out why i could not compile an emacs in docker any more.
the Problem is, that with an actual Kernel Version: 4.1.17-boot2docker,
 ```kernel.randomize_va_space = 2``` 
and emacs needs ```kernel.randomize_va_space = 0``` in order to be compiled.

would it be possible to set kernel.randomize_va_space to 0, as it was a week ago, cause this problem comes just for a couple of days up.

**Comment by AkihiroSuda** [comment]
You can set it to 0 by running  `echo 0 > /proc/sys/kernel/randomize_va_space`.
Do you think it should be zero by default?
Recent major distos seem using 2 by default.

**Comment by zavalit** [comment]
yes, i also do it that way, go and change that kernel option manually, but my problem is, that i ship the whole emacs building routine in a single [repo](https://github.com/zavalit/flink-development-suite). and it kinda not very clean solution to encourage people to go a hack a vm kernel. 

and for sure, it's still possible to provide an optional bash script that go into vm over ssh and tries to overwrite kernel.
